application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/pages/ChartAndPopupTableUI.java

```diff
@@ -223,6 +234,19 @@ public void handleEvent(Event event) {
 		scText.setExpandHorizontal(true);
 		scText.setExpandVertical(true);
 
+		scPageContainer.addListener( SWT.Resize, new Listener() {
+			@Override
+			public void handleEvent(Event event) {
+				int width = filterBar.getClientArea().width;
+				int height = timelineCanvas.getClientArea().height + filterBar.getClientArea().height
+						+ scChart.getClientArea().height;
+				if (width > 0 && height > 0) {
+					 scPageContainer.setMinSize(scPageContainer.computeSize( width, height ) );
```

Comment:
excess whitespace around the `width, height`

Reply:
fixed https://github.com/aptmac/jmc/pull/45/commits/caa3552fda290a50a95516ddb195b1de19914796